### PR TITLE
Final changes to remove expression tab and move functionality to plots

### PR DESCRIPTION
**Pull request by adamabeshouse**
Addresses points in https://github.com/cBioPortal/cbioportal/issues/7808#issue-681358551

(1) add mRNA vs Study quick plot button to Plots
(2) select log scale by default for rna-seq (for any chart)
(3) add redirection from Expression tab to Plots tab (mRNA vs Study)
(4) remove Expression tab and add an info banner about it to queries where it would have appeared.

Also fixes https://github.com/cBioPortal/cbioportal/issues/7971

image

Clicking on the link in the banner leads to -->

image

[Link](https://www.cbioportal.org/results/oncoprint?Action=Submit&RPPA_SCORE_THRESHOLD=2.0&Z_SCORE_THRESHOLD=2.0&cancer_study_list=acc_tcga_pan_can_atlas_2018%2Cchol_tcga_pan_can_atlas_2018&case_set_id=all&data_priority=0&gene_list=TP53&geneset_list=%20&plots_coloring_selection=%7B%7D&plots_horz_selection=%7B%22dataType%22%3A%22clinical_attribute%22%2C%22selectedDataSourceOption%22%3A%22CANCER_STUDY%22%7D&plots_vert_selection=%7B%22dataType%22%3A%22MRNA_EXPRESSION%22%2C%22logScale%22%3A%22true%22%7D&profileFilter=0&tab_index=tab_visualize) and run `localStorage.netlify = "deploy-preview-3430--cbioportalfrontend"`

### CODE REVIEW

**Comment by jjgao** [comment]
@adamabeshouse looks good to me. @cBioPortal/product comments?

File: src/shared/components/userMessager/UserMessage.tsx [hunk 1]
```diff
@@ -75,6 +86,12 @@ export default class UserMessager extends React.Component<
     }
 
     render() {
+        console.log(
```
Comment (alisman) [inline src/shared/components/userMessager/UserMessage.tsx:89]: console.log

File: src/shared/components/userMessager/UserMessage.tsx [hunk 2]
```diff
@@ -57,6 +67,7 @@ export default class UserMessager extends React.Component<
                 makeMessageKey(message.id)
             );
             const expired = Date.now() > message.dateEnd;
+            console.log(notYetShown, expired);
```
Comment (alisman) [inline src/shared/components/userMessager/UserMessage.tsx:70]: console.log

File: src/pages/resultsView/ResultsViewPage.tsx [hunk 3]
```diff
@@ -627,6 +627,21 @@ export default class ResultsViewPage extends React.Component<
                             />
                         </div>
                     )}
+
+                    {this.userMessages.isComplete &&
+                        this.userMessages.result.length > 0 && (
+                            <Portal
```
Comment (alisman) [inline src/pages/resultsView/ResultsViewPage.tsx:633]: would be nice if we could put portal business inside of the UserMessager component.

File: src/pages/resultsView/ResultsViewPage.tsx [hunk 4]
```diff
@@ -584,6 +539,50 @@ export default class ResultsViewPage extends React.Component<
         });
     }
 
+    readonly userMessages = remoteData({
```
Comment (alisman) [inline src/pages/resultsView/ResultsViewPage.tsx:542]: lets just add a comment here saying "this is here temporarily for orienting user to expression data in plots page" or something.

**Pull request merged**